lib/timescaledb/reporter.ex

```diff
@@ -33,24 +33,40 @@ defmodule Membrane.Telemetry.TimescaleDB.Reporter do
 
   def send_measurement(
         [:membrane, :input_buffer, :size] = event_name,
-        %{element_path: path, method: method, value: value} = measurement
+        %{element_path: path, method: method, value: value}
       )
       when is_binary(path) and is_binary(method) and is_integer(value) do
+    measurement = %{
+      element_path: extend_with_os_pid(path),
+      method: method,
+      value: value,
+      time: NaiveDateTime.utc_now()
+    }
+
     GenServer.cast(
       __MODULE__,
-      {:measurement, event_name, Map.put(measurement, :time, NaiveDateTime.utc_now())}
+      {:measurement, event_name, measurement}
     )
   end
 
   def send_measurement(
         [:membrane, :link, :new],
-        %{parent_path: parent_path, from: from, to: to, pad_from: pad_from, pad_to: pad_to} = link
+        %{parent_path: parent_path, from: from, to: to, pad_from: pad_from, pad_to: pad_to}
       )
       when is_binary(parent_path) and is_binary(from) and is_binary(to) and is_binary(pad_from) and
              is_binary(pad_to) do
+    link = %{
```

Comment:
Couldn't you just do something like `link = link |> Map.merge(%{parent_path: ..., time: ...})?  Same with measurement above

Reply:
I can do it. I just didn't know about this function